src/domain/coordinateTypeDef.js

```diff
@@ -2,6 +2,18 @@
  * @module domain/coordinateTypeDef
  */
 /**
- * An array of numbers representing an xy coordinate. Ordering is [easting, northing] or [lon, lat]. Example: `[16, 48]`.
+ * An array of numbers representing an XY coordinate. Ordering is [easting, northing] or [lon, lat]. Example: `[16, 48]`.
  * @typedef {Array<number>} Coordinate
+ * @implements {module:domain/coordinateTypeDef~CoordinateLike}
+ */
+
+/**
+ * An array of numbers or other types. The first two entries must be a number and represent an xy coordinate. Ordering is [easting, northing] or [lon, lat]. Example: `[16, 48]`.
+ * @typedef {Array<number|object>} CoordinateLike
+ */
+
+/**
+ * An array of numbers representing an XYZ coordinate. Ordering is [easting, northing, z] or [lon, lat, z]. Example: `[16, 48, 1000]`.
```

Comment:
if easting is the meaning for X and northing for Y, then elevation should be the meaning for Z. In both ordering definitions